.github/scripts/tkn_check_parse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# removes `actions's` default -e flag as it breaks the logic
+set +e
+
+setup() {
```

Comment:
Idea: What do you think about copy&pasting the installation step from the tkn push workflow (as a step)? That way both would be consistent and this script will be less complex - it seems we might not need `is_yaml` either, so we might be able to get away with functions and just have a bunch of commands. It's just too bad `TKN_CLI_VERSION` cannot be defined in a central location.

Reply:
Also, why is the output of the commands suppressed? I think it can be useful to see the output. Unless you think it's too chatty.

Reply:
Even better, we could extract the install as a local action and use it in both workflows: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-an-action-in-the-same-repository-as-the-workflow

It's an extra overhead, but I think it would be worth the value of having the version defined in a single place.

Reply:
Good idea. I will take a look at this.

Reply:
OK, great. If you don't want to dig into the local action now, I can take a stab at it later :)

Reply:
Ah, I see you're already doing it, awesome :)

Reply:
@theflockers It's looking great. Now that we have the action, can we use it also in the post-merge workflow and set the `tkn` version in the action, so that it's only defined in one place?

Reply:
Perfect!